recipes/pyregion/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyregion" %}
+{% set version = "1.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4f57070564526974661bbbf96cd1d509a708909002dd34f0354ac80c391b5e61" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (py36 and win)]
```

Comment:
Same question here as in another PR. It seems like the `numpy` versions are not identically constrained in both the `build` and `run` sections and this could possibly lead to conflicts. I'm not familiar with this package though and if this is ok.